### ISSUE TITLE
Explore order of fields re: Cost Allocation + FFP and Budget

### ISSUE THREAD

**Issue by beparticular** [18F/cms-hitech-apd#2217]
In Round 11 and 11.5 of research we noticed that folks may have been missing the table that shows the 90/10 (or 75/25, etc) split when they were expecting to see it, or may not have been understanding how and when the calculations are made there. This led to some discussions about cost allocation, other funding field placement, the order of fields on the cost allocation page and the FFP and budget page in general, the side nav labels, table design, and whether we have some opportunity for improving the understanding of this flow for users. 

Let's do a little exploration of this area to see if we can make improvements. 

Related #2169

(Refer to Caryn's content review doc as well)

### This task is done when...
 - [x] The team has the cost allocation discussion and a strong understand of what we're asking for in the APD
 - [x] A meeting is held to go over the current flow of the Cost Allocation and FFP and budget pages with Jerome, Caryn, Laura, Jenn, and Jesse, and any devs who wish to be present, learn what we can and cannot change
 - [x] We agree if any changes could be made to increase understanding and spin up issues for each change as needed (For instance explore the order of fields on the Cost Allocation and FFP and budget pages in wireframes in a separate issue)
 - [x] Designs are created to reflect the decisions made in conversations, outlined below

 - [x] Move the Other Funding field to the cost allocation page
 - [x] Create an Other Funding section to match the Cost Allocation section design. This will house the current Other Funding Description and the amount field that is moved to this page. 
- [x] The order of fields on all tables should be in the order that the information was entered. The budget summary on the FFP page is in a different order, let's fix that.  
- [ ] Rename the FFP and budget page (?) 
 - [x] Potentially rename the Cost Allocation page (Cost Allocation and Other Funding)
 - [ ] A view of the export version will need to be provided if anything will be moving around there

**Comment by beparticular** [comment]
Noted in a convo yesterday: we're going to do a full audit of the nav titles. Consider the titles of these sections are we continue this conversation.

**Comment by jjames521** [comment]
Organization and flow change plus other additives from #2235 and #2247

![SummaryBudgetByActivity.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/c4b3a8f9-1047-4fcd-bf79-1a4431005fdf)

**Comment by jjames521** [comment]
@beparticular 

Mockups that depict the new sequence for Cost Allocation and FFP and Budget page per criteria

Export view will be reviewed further from Jenn.

![CostAllocationBuilderViewVar2.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/ffd22d69-5409-4957-ab71-f3a9c30262b1)

![FFP and Budget.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/60f61ce0-6563-4613-a169-8c3590bf23b1)

For clarity on mock up I've cut off the mock at Estimated Quarterly Expenditure because no further changes in the sequence are made after that.

**Comment by beparticular** [comment]
@jjames521 from Design Sync

- For understanding the mock up - add an amount into Activity Total under Other Funding to make it clear that the math remains the same

- Switch the summary table on the FFP and budget page with the Federal state split field. The table informs the decisions on this page, but isn’t changed by anything entered on this page. 

- Once those changes are made, tag me and @jeromeleecms for one last look before moving to dev. Thx!

**Comment by beparticular** [comment]
This looks good to me. @jeromeleecms any last thoughts? We'd like to close this out this sprint.

**Comment by jeromeleecms** [comment]
@beparticular @jjames521 Are we asking the Dev's to put in placeholder text? It might make it easier for the folks putting in content if we can just replace that within the YAML and not ask the Dev's to do it for us.

If the answer is yes, we should include both instruction text and help text for:
1) Other Funding Amounts (we already have #2281)
2) Federal-State Split (add instruction text + help text is already present) (#2283)

If the answer is no, for the final version, let's remove the sample text for now.

Also, we had discussed replacing the existing Summary Budget table with the version from the Summary Budget Table by Activity, but I don't see an issue tracking that. Did you want to update the mock-up with that version and we can split this design task into two dev tasks (one for the field order and one for replacing the budget table)?

**Comment by beparticular** [comment]
@pphillips-fearless what would you prefer here? would you like us to note that this ticket is blocked by the content tix and do it all at one time, or is putting in placeholder text ok? 

@jeromeleecms we should add a ticket in new tix to track that issue. We don't need to change the mock up to reflect the new table - I think, but might defer to Paul on that as well.

**Comment by jjames521** [comment]
Just to recap on this

@beparticular @CSwartzHMA 

I spoke with @pphillips-fearless and he would prefer content already fleshed out so, blocked by content tix #2283 for the time being.

**Comment by jjames521** [comment]
New mockups per the change in content

@beparticular

![CostAllocationBuilderViewVar2Edit1.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/6fe45a49-a119-4dc9-a291-57c97574810d)

![FFP and BudgetEdit1.png](https://images.zenhubusercontent.com/5e7151677c78c820a2a11970/216473e8-e43b-49f1-94e5-d088c5020ca4)

**Comment by CSwartzHMA** [comment]
I think this is so much more clear and the flow of information is much smoother.

**Comment by beparticular** [comment]
@pphillips-fearless the help text we were waiting on for this ticket has been added to the mockups and this is ready to be a dev ticket. The summary and mockups are in the last comment Jesse made here.

**Comment by pphillips-fearless** [comment]
new dev task is #2322
